crowdkit/datasets/load_dataset.py

```diff
@@ -31,6 +31,6 @@ def load_dataset(dataset: str, data_dir: Optional[str] = None) -> Tuple[pd.DataF
 
 
 def get_datasets_list() -> List[Tuple[str, str]]:
-    """Returns a list of available datasets in format [(name, description)]."""
+    """Returns a list of available [datasets](https://toloka.ai/datasets) in format [(name, description)]."""
```

Comment:
Same as above in `load_dataset()`, good job.

Reply:
No, see above

Reply:
Agreed.